named_entity_recognition/documentation/step-2-build-components.md

```diff
@@ -0,0 +1,75 @@
+# Build components
+
+A component is code that performs one step in the Kubeflow pipeline. It is a containerized implementation of an ML task. **Components can be reused in other pipelines.**
+
```

Comment:
This section could benefit from a diagram or explicit callout of the pieces involved in using a component:

- Container image
- Yaml specification
- Python reference in the pipeline itself

Reply:
✓ done (added more information in step 5 due to context)